CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read along to learn how to get started. More information can be found in each of the README.md as well as  https://docs.airswap.io/ about the individual repos.
```

Comment:
`Please read along...` -> `Please read this guide...`

Reply:
'More information' - about what?

Reply:
`More information about the individual sub-repos can be found in the relevant README.md, as well as on the [airswap docs site](https://docs.airswap.io/)`